packages/shared-ux/button/exit_full_screen/src/__snapshots__/exit_full_screen_button.test.tsx.snap

```diff
@@ -46,9 +46,92 @@ exports[`<ExitFullScreenButton /> with kibana services is rendered 1`] = `
         <EuiScreenReaderOnly>
           <p
             aria-live="polite"
-            className="euiScreenReaderOnly"
+            css="unknown styles"
+            key="null"
```

Comment:
I would like to understand why `css` and `key` are appearing here with unknown/null values? This doesn't seem right.

Reply:
Agreed, this doesn't seem right. This looks like something in a build step failed. I'm investigating now.

Reply:
This is (unfortunately) correct as far as component output when using the `css` prop with Emotion and taking a snapshot of an Enzyme `mount` call.
For instances like this calling `.render()` will give more reliable output.